src/controllers/dashboard/devices/devices.html

```diff
@@ -4,8 +4,15 @@
             <div class="verticalSection verticalSection">
                 <div class="sectionTitleContainer sectionTitleContainer-cards flex align-items-center">
                     <h2 class="sectionTitle sectionTitle-cards">${HeaderDevices}</h2>
-                    <a is="emby-linkbutton" rel="noopener noreferrer" class="raised button-alt headerHelpButton" target="_blank" href="https://jellyfin.org/docs/general/server/devices">${Help}</a>
-                    <button id="deviceDeleteAll" is="emby-button" type="button" class="raised button-alt headerHelpButton">${DeleteAll}</button>
+                    <button
+                        id="deviceDeleteAll"
+                        is="emby-button"
+                        type="button"
+                        class="raised button-alt"
+                        style="margin-left: 1.25em !important; padding-bottom: 0.4em !important; padding-top: 0.4em !important;"
```

Comment:
Shouldn't style attributes be avoided or is this common in React dev 👀?

Reply:
Yeah that's fair... this is not ideal, **but** I have a branch porting this page to React locally and I didn't want to add a CSS class that I'm just going to remove soon. :sweat_smile:

Reply:
Yeah fair enough!